_content/news/_posts/2018-12-28_lizardxmas2018.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "Lizards naar X-Mas Tournament in Kortrijk"
+date: 2018-12-11 12:00:00
+description: De Limburg Lizards doen mee aan dit internationale jeugdbasketbaltoernooi.
+cover: /news/img/LizardTraining_A.jpg
+permalink: /news/2018-12-28-lizardsxmas2018/
```

Comment:
de permalink is 2018-12-2018-lizardsxmas2018 maar de files is 2018-12-2018_lizardsxmas2018. Ik stel voor dat je in de filename de underscore vervangt door een streepje